vncviewer/parameters.cxx

```diff
@@ -101,10 +101,13 @@ IntParameter qualityLevel("QualityLevel",
                           8);
 
 BoolParameter maximize("Maximize", "Maximize viewer window", false);
-BoolParameter fullScreen("FullScreen", "Full screen mode", false);
+BoolParameter fullScreen("FullScreen", "Enable full screen", false);
+StringParameter fullScreenMode("FullScreenMode", "Specify what mode to use when full screen is enabled. "
+                                                 "Should be either Current or All",
+                                                 "Current");
 BoolParameter fullScreenAllMonitors("FullScreenAllMonitors",
-                                    "Enable full screen over all monitors",
-                                    true);
+                                    "[DEPRECATED] Enable full screen over all monitors",
+                                    false);
```

Comment:
Did you test `removeParam()`?

Reply:
`removeParam()` results in it being reject on command line too. We will keep it for now.